### PR TITLE
#364 Tools | Options | Text Editor | Python | Formatting | Spacing can't be loaded

### LINKED ISSUE #364: Tools | Options | Text Editor | Python | Formatting | Spacing can't be loaded
VS 2015 RC 23008
New Python Web project
Tools | Options | Text Editor | Python | Formatting | Spacing 

VS shows 'an error loading property page'. Same with Statements and Wrapping. General appears to be OK.

### PR DESCRIPTION
**Pull request by zooba**
Fixes #364 Tools | Options | Text Editor | Python | Formatting | Spacing can't be loaded
Removes folder images from treeview.
Also prevents IntelliSense providers from loading against the preview editor to avoid many assertions.

### CODE REVIEW

File: Python/Product/PythonTools/PythonTools/Intellisense/SmartTagControllerProvider.cs [hunk 1]
```diff
@@ -22,23 +22,25 @@
 namespace Microsoft.PythonTools.Intellisense {
 #if !DEV14_OR_LATER
     [Export(typeof(IIntellisenseControllerProvider)), ContentType(PythonCoreConstants.ContentType), Order]
+    [TextViewRole(PredefinedTextViewRoles.Analyzable)]
+    [TextViewRole(PredefinedTextViewRoles.Editable)]
     class SmartTagControllerProvider : IIntellisenseControllerProvider {
 
-        #region MEF Imports
+    #region MEF Imports
```
Comment (zooba) [inline Python/Product/PythonTools/PythonTools/Intellisense/SmartTagControllerProvider.cs]: Removing these regions. We don't need two regions for a three line class...

File: Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs [hunk 2]
```diff
@@ -42,28 +39,16 @@ class OptionsTreeView : TreeView {
...
-                    using (var g = Graphics.FromImage(bmp)) {
-                        g.FillRectangle(
-                            Brushes.Magenta,
-                            new Rectangle(0, 0, 16, 16)
-                        );
-                    }
-                    _imageList.Images.Add(bmp);
-                    _imageList.Images.Add(openFolder);
-                    _imageList.Images.Add(closedFolder);
+                _imageList = new ImageList();
```
Comment (huguesv) [inline Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs]: Is it a problem if StateImageList is initialized to null?  If not, we could remove _imageList completely.

Comment (zooba) [inline Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs]: I think we need the empty image to get proper layout - don't really want to dig deep enough to figure out how to get that layout without it, and frankly I'd rather just use a WPF treeview there so we don't have to do the ownerdrawn stuff at all. But that's for a future release.

Right now, _imageList is static and that's how we avoid creating lots of image lists over the life of the process.

Comment (huguesv) [inline Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs]: Oh I missed the static there, yeah, better not change it.  I agree about WPF treeview.  I know the high dpi support on these formatting options pages has been problematic at 200%+ and WPF.  I haven't ported the issues from codeplex as I don't have access to a machine that can reasonably go that high anymore.

**Comment by huguesv** [comment]
lgtm

File: Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs (hunk 2 above)
Comment (zooba) [inline Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs]: My laptop does, and I can confirm that it's still an issue :)

**Pull request merged**